client/src/shared/Navbar.tsx

```diff
@@ -1,34 +1,114 @@
 import React from "react";
 import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Grid from "@material-ui/core/Grid";
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
 
 import {
     BlackTextTypography,
-    Link
+    SecondaryMainTextTypography,
+    Link,
+    Button,
   } from "../components/index";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    width: "90%",
-    padding: "0.5% 5% 0.5% 5%",
+    padding: "2% 6% 0.5% 6%",
     background: "white",
     position: "fixed",
     marginTop: "-8px",
-    zIndex: 5
+    zIndex: 999, 
+    fontSize: "0.9em",
+    boxShadow: "none"
   },
-  tabs: {
-    backgroundColor: theme.palette.primary.light,
-    boxShadow: "0",
-    paddingTop: "13px",
+  tab: {
+    margin:"0 10%", 
+    display:"inline-block"
   },
 }));
 
 export default function Navbar(props: any) {
+  const [anchorEl, setAnchorEl] = React.useState(null);
   const classes = useStyles();
 
+  let name = "";
+  if (props.user) {
+    name = props.user.firstName + " " + props.user.lastName;
+  }
+
+  const logout = () => {
+    localStorage.clear();
+    window.location.reload();
+    setAnchorEl(null);
+  }
+
+  const handleClick = (event: any) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const content = () => {
+    let userType = localStorage.getItem("userType");
+    // admin
+    if (userType === "0"){
+      return <Grid style={{margin:"0 15%"}}>
+        <Link href="/events" className={classes.tab}><SecondaryMainTextTypography>Dashboard</SecondaryMainTextTypography></Link>
+        <Link href="/volunteers" className={classes.tab}><SecondaryMainTextTypography >Browse Volunteers</SecondaryMainTextTypography></Link>
+        <Link className={classes.tab}><SecondaryMainTextTypography>Admin</SecondaryMainTextTypography></Link>
+      </Grid>;
+    } else if (userType === "1") {
```

Comment:
There's an enum in `client/src/data/types/userTypes.ts` called `UserType` that can be used for these comparisons

Reply:
although it's missing a type for `Admin`, so feel free to add that :)